### DISCUSSION TITLE
测试框架 PyMilvus使用

### DISCUSSION THREAD

**Discussion by zs001122** [milvus-io/milvus#37355] [Q&A and General discussion] [answered]
![image](https://github.com/user-attachments/assets/581539cf-37ec-402f-92f5-ab14cb3925a5)
这个是版本问题吗？我测了前后几个都出现这个问题

**Comment by yhmo** [comment, marked as answer]
FunctionType 是 2.5的功能，只在pymilvus master分支上有，你如果用2.4的pymilvus来测试，就不会有这个类

**Reply by zs001122** [reply]
好的，谢谢
